pkg/kubelet/cm/devicemanager/endpoint.go

```diff
@@ -87,7 +87,9 @@ func (e *endpointImpl) getPreferredAllocation(available, mustInclude []string, s
 	if e.isStopped() {
 		return nil, fmt.Errorf(errEndpointStopped, e)
 	}
-	return e.api.GetPreferredAllocation(context.Background(), &pluginapi.PreferredAllocationRequest{
+	ctx, cancel := context.WithTimeout(context.Background(), pluginapi.KubeletGetPreferredAllocationRPCTimeoutInSecs*time.Second)
+	defer cancel()
+	return e.api.GetPreferredAllocation(ctx, &pluginapi.PreferredAllocationRequest{
```

Comment:
do we have test coverage for this change?

Reply:
thanks for the reminder. how can i do test coverage for this change?